cmd/vm_start.go

```diff
@@ -54,10 +63,16 @@ func startVirtualMachine(vmName string) error {
 		return fmt.Errorf("%q is not in a %s state, got: %s", vmName, state, vm.State)
 	}
 
-	_, err = asyncRequest(&egoscale.StartVirtualMachine{ID: vm.ID}, fmt.Sprintf("Starting %q ", vm.Name))
+	cmd := &egoscale.StartVirtualMachine{ID: vm.ID}
+	if (vmRescueProfile != nil) {
+		cmd.RescueProfile = *vmRescueProfile
+	}
+
+	_, err = asyncRequest(cmd, fmt.Sprintf("Starting %q ", vm.Name))
 	return err
 }
 
 func init() {
+	vmStartCmd.Flags().StringP("rescue-profile", "P", "", "option rescue profile when starting VM")
```

Comment:
Captain Nitpick here: I'm not sure about the `-P` short flag. To me short flags should be reserved for global flags (e.g. help, config, verbose, version, account, debug...).

Reply:
can I prevent short flags by supplying an empty string here?

Reply:
The doc says yes, I'll amend

Reply:
Yes.